File: C/medidas.c
#include <stdio.h>

int main() {

    double a, b, c, areaQuadrado, areaTriangulo, areaTrapezio;

    printf("Digite a medida A: ");
    scanf("%lf", &a);
    printf("Digite a medida B: ");
    scanf("%lf", &b);
    printf("Digite a medida C: ");
    scanf("%lf", &c);

    areaQuadrado = a * a;
    areaTriangulo = (a * b) / 2;
    areaTrapezio = (a + b) * c / 2;

    printf("AREA QUADRADO = %.4lf", areaQuadrado);
    printf("\nAREA TRIANGULO = %.4lf", areaTriangulo);
    printf("\nAREA TRAPEZIO = %.4lf", areaTrapezio);

    return 0;
}

File: Java/medidas.java
import java.util.Locale;
import java.util.Scanner;

public class medidas {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double a, b, c, areaQuadrado, areaTriangulo, areaTrapezio;

        System.out.print("Digite a medida A: ");
        a = sc.nextDouble();
        System.out.print("Digite a medida B: ");
        b = sc.nextDouble();
        System.out.print("Digite a medida C: ");
        c = sc.nextDouble();

        areaQuadrado = a * a;
        areaTriangulo = (a * b) / 2;
        areaTrapezio = (a + b) * c / 2;

        System.out.println("AREA DO QUADRADO = " + String.format("%.4f", areaQuadrado));
        System.out.println("AREA DO TRIANGULO = " + String.format("%.4f", areaTriangulo));
        System.out.println("AREA DO TRAPEZIO = " + String.format("%.4f", areaTrapezio));

        sc.close();
    }
}